framework/wazuh/core/wdb.py

```diff
@@ -233,7 +238,7 @@ def send_request_to_wdb(query_lower, step, off, response):
                 lim = int(re.compile(r".* limit (\d+)").match(query_lower).group(1))
                 query_lower = query_lower.replace(" limit {}".format(lim), "")
 
-            regex = re.compile(r"\w+(?: \d*|)? sql select ([A-Z a-z0-9,*_` \.\-%\(\):\']+) from")
+            regex = re.compile(r"\w+(?: \d*|)? sql select ([A-Z a-z0-9,*_` \.\-%\(\):\']+?) from")
```

Comment:
What caused this change?

Reply:
In queries like the ones created for Rootcheck, the regex did not work well since it matched everything from `sql select` to the last `from`. Original query:

```SQL
agent 000 sql select pci_dss as 'pci',date_last as 'readDay',date_first as 'oldDay',status as 'status',cis as 'cis',log as 'event' from (select 'outstanding' as status, date_first, date_last, log, pci_dss, cis from pm_event as t where date_last > (select date_last-86400 from pm_event where log = 'Ending rootcheck scan.') union select 'solved' as status, date_first, date_last, log, pci_dss, cis from pm_event as t where date_last <= (select date_last-86400 from pm_event where log = 'Ending rootcheck scan.')) where log not in ('Starting rootcheck scan.', 'Ending rootcheck scan.', 'Starting syscheck scan.', 'Ending syscheck scan.') and (pci_dss = 'all' collate nocase) and (cis = 'all' collate nocase)   order by date_last desc
```

**Using original regex:**
```
re.compile(r"\w+(?: \d*|)? sql select ([A-Z a-z0-9,*_` \.\-%\(\):\']+) from")
```
It would match beyond first `from`:
```SQL
pci_dss as 'pci',date_last as 'readDay',date_first as 'oldDay',status as 'status',cis as 'cis',log as 'event' from (select 'outstanding' as status, date_first, date_last, log, pci_dss, cis from
```

**Adding `?` to the regex (lazy match):**
```
re.compile(r"\w+(?: \d*|)? sql select ([A-Z a-z0-9,*_` \.\-%\(\):\']+?) from")
```
It matches only until the first `from` is found, which from my point of view, is the expected behaviour:
```SQL
pci_dss as 'pci',date_last as 'readDay',date_first as 'oldDay',status as 'status',cis as 'cis',log as 'event' from
```